content-src/components/DiscoveryStreamComponents/DSCard/DSCard.jsx

```diff
@@ -9,6 +9,34 @@ import { ImpressionStats } from "../../DiscoveryStreamImpressionStats/Impression
 import React from "react";
 import { SafeAnchor } from "../SafeAnchor/SafeAnchor";
 
+const DefaultMeta = ({ source, title, excerpt, context, isPromo }) => (
+  <div className="meta">
+    <div className="info-wrap">
+      <p className="source clamp">{source}</p>
+      <header className="title clamp">{title}</header>
+      {excerpt && <p className="excerpt clamp">{excerpt}</p>}
+      {isPromo && <span className="learn-more">Learn More</span>}
+    </div>
+    {context && <p className="context">{context}</p>}
+  </div>
+);
+
+const VariantMeta = ({ source, title, excerpt, context, isPromo }) => (
+  <div className="meta">
+    <div className="info-wrap">
+      <p className="source clamp">
+        {source}
+        {context && <span className="context top">&#8226; Sponsored</span>}
+      </p>
+      <header className="title clamp">{title}</header>
+      {excerpt && <p className="excerpt clamp">{excerpt}</p>}
+      {isPromo && (
+        <button className="button learn-more-button">Learn More</button>
```

Comment:
Is the button tied to being a promo, or is just just a button we can use for any cta/link?

Reply:
Its tied to being a promo. We don't show button for publisher content in card. Clarified with @wolasi button is linked to same url as the card_url